src/client/index.tsx

```diff
@@ -41,26 +45,26 @@ class Page extends Component<Props, State> {
   }
 }
 
+// Header
+let pageTitle = (document.getElementById('header').getAttribute('pageTitle'))
```

Comment:
These elements can be null (`document.getElementById('header')` and `document.getElementById('react-header')`). If they are null, then you probably get an ugly error. You want to guard for this case with something like:
```
let pageTitle = '';
const pageTitleElm = document.getElementById('header');
if (pageTitleElm) {
  pageTitle = pageTitleElm.getAttribute('pageTitle') || '';
}
```